.github/workflows/lint.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    name: 🧹 Markdown and YAML Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: yaml-lint
+        uses: ibiqlik/action-yamllint@v1
```

Comment:
Please move `yamllint.yml` from the root folder to `.github/workflows/` and use [config_file](https://github.com/ibiqlik/action-yamllint#optional-parameters) to specify it.
The idea is to not pollute the root folder with too many config files which are used for very specific needs.